schedulearn-frontend/src/components/NestedList/NestedList.tsx

```diff
@@ -219,6 +219,7 @@ export class NestedList<TItem extends ListItem<TItem>>
                 index={index}
                 callback={this.onListItemClick}
                 infoCallback={this.onInfoItemClick}
+                disabled={this.props.disabled}
```

Comment:
You forgot to disable back button (line 207 in NestedList). I recommend using logic (showButton && !disabled), so that it would return undefined if it's disabled or the button does not need to be shown